.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
Well I checked them out. Airbnb uses much of a react based linting style. Google's eslint config seems to be good. But I would still prefer eslint recommended style since others need some extra dev dependencies.